install_config/install/deploy_router.adoc

```diff
@@ -484,6 +549,72 @@ $ modshard 3 \
     ROUTE_LABELS='sla=low'
 ----
 
+[[using-namespace-shards]]
+=== Using Namespace Shards
+
+Each
+link:../../architecture/core_concepts/projects_and_users.html#projects[project has its own namespace].
+The routes for a project can be handled by a selected router by using NAMESPACE_LABELS.
+The router is given a NAMESPACE_LABELS label and each project that uses the router is
+also given the same NAMESPACE_LABELS label.
+
+Because NAMESPACE_LABELS labels are used, the router service account associated with the
+router must have `cluster-reader` permission.  Multiple routers can be deployed using the
+same service account.
+
+Add `cluster-reader` to the router service account, in this case the default route:
+====
+----
+$ oadm policy add-cluster-role-to-user cluster-reader system:serviceaccount:default:router
+----
+====
+Now create and label the router:
```

Comment:
We aren't labelling it... we are providing it with a label selector.  The labels are on the namespaces.